### ISSUE TITLE
Accessibility feedback mechanism

### ISSUE THREAD

**Issue by yaelberger-commits** [cds-snc/notification-planning#1018]
_Accessibility feedback mechanism_

## Problem statement
As an authenticated GC Notify government client, I need to be able to submit feedback on accessibility issues and barriers with Notify so that I can use Notify easily without barriers and so I can help improve the product for others. 

### **WHY** are we building?
- new Canada.ca accessibility mechanism pattern 
- https://docs.google.com/presentation/d/1A9s7r4TpSSOg0Ik65l2nf6RQeTO-6A75aLoK6llINRY/edit#slide=id.g198b545baa4_0_13

### **WHAT** are we building?
An accessibility specific feedback form 
https://forms-formulaires.alpha.canada.ca/id/clk1jv0jo00skyq65po4159gw

### **VALUE** created by our solution
- compliance with canada.ca style
- More ways for folks experiencing barriers to let us know
- More cohesive and uniform experience across government websites

## Acceptance Criteria** (Definition of done) 
Given some context, when (X) action occurs, then (Y) outcome is achieved 
- [ ] The a11y feedback form can only be accessed by Notify clients who are signed-in to the platform
- [ ] A documented procedure exists for the team on how to handle private information generated by this form
- [ ] Generate appropriate log messages so that executions of this feature can be tracked
- [ ] Can misuse of this feature cause harm? If yes, create an alert

* A11y
* Bilingualism
* Privacy considerations

### Design sizing on this issue:
**Service Design:** Small, known-unknown issue
- The scope is clear, we require mapping and flows to establish the service journey and workflows

**UX: Small Known-Known issue**
- There are many examples of accessibility feedback mechanisms. 
- We require independent pattern research and UI design based on established patterns

**Content: Small Known-Known issue**
- There are many examples of accessibility feedback mechanisms. 
- We require content alignment with GC & Notify content practices 

### This is primarily a Service Design issue:
Service design will lead, UX and Content will provide supporting activities

## QA Steps
- [ ] Tested in a realistic production scenario

**Comment by YedidaZalik** [comment]
@yaelberger-commits could this go on the page with the accessibility statement or on the contact us page?
(or both with an all doors are the right door approach?)

**Comment by yaelberger-commits** [comment]
@YedidaZalik yes could be accessed from both doors, but first we have to build a feedback form for accessibility feedback

**Comment by YedidaZalik** [comment]
@yaelberger-commits I'm wondering if we need a separate form. I'm thinking that we could add an accessibility feedback option to the contact us page and link to that from the accessibility statement.

**Comment by YedidaZalik** [comment]
Possibly for next sprint

**Comment by YedidaZalik** [comment]
Could bring to refinement this week
Mohamed to lead story refinement on this card

**Comment by YedidaZalik** [comment]
So that we remember, a<img width="298" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK3k5QkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--45dbfeffeb10d11c5cd765f8bb37320f846c4613/Screenshot%202024-05-07%20at%209.33.14%E2%80%AFAM.png" alt="Screenshot 2024-05-07 at 9.33.14 AM.png" />
dding screenshots of discussion about this in Ally stat<img width="375" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBKzI5QkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--b2abdd90c43eafe68b187732d241b72d7e933ea3/Screenshot%202024-05-07%20at%209.33.25%E2%80%AFAM.png" alt="Screenshot 2024-05-07 at 9.33.25 AM.png" />
ement comments <img width="272" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBKzY5QkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--8e583cfe0aa2a5810ddabc881d5551f1bdb9ec7c/Screenshot%202024-05-07%20at%209.33.34%E2%80%AFAM.png" alt="Screenshot 2024-05-07 at 9.33.34 AM.png" />
<img width="264" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBKys5QkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--342fa69fac568b6e43aade3fb2bd40614175ff5e/Screenshot%202024-05-07%20at%209.34.16%E2%80%AFAM.png" alt="Screenshot 2024-05-07 at 9.34.16 AM.png" />
<img width="318" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL0M5QkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--53bee338e81a39160cef8d76ad78d788dc9d4499/Screenshot%202024-05-07%20at%209.32.54%E2%80%AFAM.png" alt="Screenshot 2024-05-07 at 9.32.54 AM.png" />

**Comment by yaelberger-commits** [comment]
Yael, Melissa and Andréanne will meet this week to discuss taking a Platform approach to a a11y feedback mechanism

**Comment by yaelberger-commits** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/1018?planning-poker) with Zenhub @amazingphilippe @YedidaZalik @mtoutloff

**Comment by yaelberger-commits** [comment]
Using GC Forms to collect the info would be more private, could then download as CSV on ESDC device

**Comment by schancase** [comment]
notes from feature refinement today with @connorscarolyns and @andrewleith  (for Jumana) - we're going to set this feature up as a single end-to-end feature that encompases: service design flows, plus design + dev implementation. Thinking all this should be doable within 2 sprints. cc @yaelberger-commits, @jzbahrai

**Comment by schancase** [comment]
removed estimate (13) as we are no longer using these

**Comment by schancase** [comment]
This was an unparented Epic. I have just parented to our Q4 Initiative, assuming we plan to close this in this current Quarter. If not, it can be re-parented to a future FY Quarter. cc @connorscarolyns

**Comment by schancase** [comment]
this is very close to closing - associating with sprint 18

**Comment by connorscarolyns** [comment]
Closed as complete!!!!! Great job everyone
